crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -468,7 +460,9 @@ def services
           false,
           services: {
             data: e.message,
-            help: "Crowbar has failed. Check /var/log/crowbar/production.log for details."
+            help: "Check /var/log/crowbar/production.log at admin server for details." \
+              "If the action failed at a specific node, " \
```

Comment:
nit: Actually I meant `s/at/on ` for all the occurrences here. (And in this PR in general).

Reply:
Oh. Sure, I missed the other one :-)